documentation/Installation.md

```diff
@@ -6130,3 +6130,33 @@ The tables below shows the correspondence of versions that are supported and is
 |          | kubernetesui/dashboard                                         | v2.7.0           | v2.7.0                       | v2.7.0       | v2.7.0       | v2.7.0            | v2.7.0    | v2.7.0    | Required only if Kubernetes Dashboard plugin is set to be installed. |
 |          | kubernetesui/metrics-scraper                                   | v1.0.8           | v1.0.8                       | v1.0.8       | v1.0.8       | v1.0.8            | v1.0.8    | v1.0.8    | Required only if Kubernetes Dashboard plugin is set to be installed. |
 |          | rancher/local-path-provisioner                                 | v0.0.31          | v0.0.31                      | v0.0.31      | v0.0.31      | v0.0.31           | v0.0.31   | v0.0.31   | Required only if local-path provisioner plugin is set to be installed. |   
+
+## Default Dependent Components Versions for Kubernetes Versions v1.34.0
```

Comment:
Probably need to plan two things in future:
* upgrade calico to v3.30.x, since 3.29 is not tested on k8s 1.33+
* change default k8s version to v1.33

Reply:
Okay will remember and plan that in future PR

Reply:
Also need to update calico for k8s 1.33